modules/dr-restoring-cluster-state.adoc

```diff
@@ -38,14 +38,16 @@ This procedure assumes that you copied the `snapshot_db_kuberesources_<datetimes
 [core@ip-10-0-143-125 ~]$ export INITIAL_CLUSTER="etcd-member-ip-10-0-143-125.ec2.internal=https://etcd-0.clustername.devcluster.openshift.com:2380,etcd-member-ip-10-0-35-108.ec2.internal=https://etcd-1.clustername.devcluster.openshift.com:2380,etcd-member-ip-10-0-10-16.ec2.internal=https://etcd-2.clustername.devcluster.openshift.com:2380"
 ----
 
+.. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+
```

Comment:
A minor concern is user don't know how to check cluster-wide proxy is enabled or not in cluster, so the perfect doc may think about adding some tips here, I made an example below, pls feel free to prune it.

# oc get proxy cluster -o yaml
apiVersion: config.openshift.io/v1
kind: Proxy
metadata:
  creationTimestamp: "2020-01-15T03:07:03Z"
  generation: 1
  name: cluster
  resourceVersion: "431"
  selfLink: /apis/config.openshift.io/v1/proxies/cluster
  uid: 1ab94060-3744-11ea-8384-xxxxxxxx
spec:
  httpProxy: http://proxy-user1:JYgU8qRZV4DY4PXJbxJK@ec2-3-135-249-22*.us-east-2.compute.amazonaws.com:3128
  httpsProxy: http://proxy-user1:JYgU8qRZV4DY4PXJbxJK@ec2-3-135-249-22*.us-east-2.compute.amazonaws.com:3128
  noProxy: test.no-proxy.com
  trustedCA:
    name: ""
status:
  httpProxy: http://proxy-user1:JYgU8qRZV4DY4PXJbxJK@ec2-3-135-249-22*.us-east-2.compute.amazonaws.com:312x
  httpsProxy: http://proxy-user1:JYgU8qRZV4DY4PXJbxJK@ec2-3-135-249-22*.us-east-2.compute.amazonaws.com:312x
  noProxy: .cluster.local,.svc,.us-east-2.compute.internal,10.0.0.0/16,10.128.0.0/14,127.0.0.1,169.254.169.254,172.30.0.0/16,api-int.qe-hasha-415.qe.devcluster.openshift.com,etcd-0.qe-hasha-415.qe.devcluster.openshift.com,etcd-1.qe-hasha-415.qe.devcluster.openshift.com,etcd-2.qe-hasha-415.qe.devcluster.openshift.com,localhost,test.no-proxy.com

Reply:
@geliu2016 Thanks for this. I can definitely add info on how to verify whether it is enabled or not, though I would prefer not to put a yaml code block. Could we say something like "You can check whether it is enabled by executing `oc get proxy cluster -o yaml`" and ...

... so it doesn't seem like there is one particular field to check whether it's enabled or not. Is it true to say that it's enabled if the `httpProxy`, `httpsProxy`, and `noProxy` fields have values set? Or is there a better way to check?

Reply:
I pushed an update with a TIP note if you want to see whether this looks okay (if the above statement I proposed is indeed true).

Reply:
ok,  there is another way in my knowledge scope would be better, "#oc describe proxy cluster":

**1).http_proxy enabled env:**

# oc describe proxy cluster
Name:         cluster
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  config.openshift.io/v1
Kind:         Proxy
Metadata:
  Creation Timestamp:  2020-01-17T08:06:02Z
  Generation:          1
  Resource Version:    433
  Self Link:           /apis/config.openshift.io/v1/proxies/cluster
  UID:                 d******-dc8d-44f1-b****************
Spec:
  Http Proxy:   http://proxy-user1:JY*************K@ec2-3-15-22-.us-east-2.compute.amazonaws.com:3128
  Https Proxy:  http://proxy-user1:J*****************K@ec2-3-15-22-*.us-east-2.compute.amazonaws.com:3128
  No Proxy:     test.no-proxy.com
  Trusted CA:
    Name:  
Status:
  Http Proxy:   http://proxy-user1:JY**************K@ec2-3-15-22-130.us-east-2.compute.amazonaws.com:3128
  Https Proxy:  http://proxy-user1:J************K@ec2-3-15-22-130.us-east-2.compute.amazonaws.com:3128
  No Proxy:     .cluster.local,.svc,.us-east-2.compute.internal,10.0.0.0/16,10.128.0.0/14,127.0.0.1,169.2xx.1xx.254,172.30.0.0/16,api-int.geliu-proxy.qe.devcluster.openshift.com,etcd-0.geliu-proxy.qe.devcluster.openshift.com,etcd-1.geliu-proxy.qe.devcluster.openshift.com,etcd-2.geliu-proxy.qe.devcluster.openshift.com,localhost,test.no-proxy.com
Events:         <none>

**2). Disabled http_proxy**

# oc get proxy cluster
NAME      AGE
cluster   3h43m
# oc describe proxy cluster
Name:         cluster
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  config.openshift.io/v1
Kind:         Proxy
Metadata:
  Creation Timestamp:  2020-01-17T05:49:57Z
  Generation:          1
  Resource Version:    427
  Self Link:           /apis/config.openshift.io/v1/proxies/cluster
  UID:                 98e4****-f********3-b****e-0**********f
Spec:
  Trusted CA:
    Name:  
Status:
Events:  <none>

Reply:
and the export command line should match with proxy: cluster description:

export HTTP_PROXY=http://proxy-user1:JY**************K@ec2-3-*-*-*.us-east-2.compute.amazonaws.com:3128
export HTTPS_PROXY= http://proxy-user1:J*****************K@ec2-3-*-*-*.us-east-2.compute.amazonaws.com:3128
$ export NO_PROXY=test.no-proxy.com

I tried it on 4.3 cluster, after export this env variables, backup/restore operation working well

Reply:
@geliu2016 are you saying that we should add in a step to have them export these 3 proxy variables? (Instead of assuming that they already have them set?)

So, we'd have them first check that it's enabled by running "oc describe proxy cluster". And if it's not (as in they do NOT see values in the description for those 3 variables), then they would proceed to the next step. And if it is enabled (as in they DO see values in the description for those 3 variables), then we would have them export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY?